python/dolfinx/io.py

```diff
@@ -56,3 +60,220 @@ def read_mesh(self, ghost_mode=cpp.mesh.GhostMode.shared_facet, name="mesh", xpa
         mesh._ufl_domain = domain
 
         return mesh
+
+
+def gmsh_to_dolfin(gmsh_cell):
```

Comment:
I think this should be in C++.

Reply:
I don't know. Do many people interface gmsh through C++ (Seems unnecessary to place it in C++ if it is not used there)?
And then there will be a lot of duplication of the mesh ordering if we would like to expose that to the user in both python and C++

Reply:
Why would there be duplication? We should just be handling gmsh and VTK in the same way.

Reply:
The duplication I am thinking of is the node ordering for each dolfin element. I thought it would be best to have this in the python layer, which is then again exposed in the python docs.
We can of course expose it just through the C++ docs instead.

Reply:
If it's in c++ it is (1) consistent with the the VTK, and (2) is available from C++ with basically no extra work.